strawberryfields/ops.py

```diff
@@ -966,16 +1352,77 @@ def _decompose(self, reg, **kwargs):
 
 
 class Sgate(Gate):
-    r"""Phase space :ref:`squeezing <squeezing>` gate.
+    r"""Phase space squeezing gate.
 
     .. math::
-       S(z) = \exp\left(\frac{1}{2}(z^* a^2 -z {a^\dagger}^2)\right)
+        S(z) = \exp\left(\frac{1}{2}(z^* a^2 -z {a^\dagger}^2)\right)
 
     where :math:`z = r e^{i\phi}`.
 
     Args:
         r (float): squeezing amount
         phi (float): squeezing phase angle :math:`\phi`
+
+    .. details::
+
+        .. admonition:: Definition
+            :class: defn
+
+            .. math::
+                & S(z) = \exp\left(\frac{1}{2}\left(z^* \a^2-z {\ad}^{2} \right) \right)
+                = \exp\left(\frac{r}{2}\left(e^{-i\phi}\a^2 -e^{i\phi}{\ad}^{2} \right) \right)\\
+                & S^\dagger(z) \a S(z) = \a \cosh(r) -\ad e^{i \phi} \sinh r\\
+                & S^\dagger(z) \ad S(z) = \ad \cosh(r) -\a e^{-i \phi} \sinh(r)
+
+            where :math:`z=r e^{i \phi}` with :math:`r \geq 0` and :math:`\phi \in [0,2 \pi)`.
+
+        The squeeze gate affects the position and momentum operators as
+
+        .. math::
+            S^\dagger(z) \x_{\phi} S(z) = e^{-r}\x_{\phi}, ~~~ S^\dagger(z) \p_{\phi} S(z) = e^{r}\p_{\phi}
+
+        The Fock basis decomposition of displacement and squeezing operations was analysed
+        by Krall :cite:`kral1990`, and the following quantity was calculated,
+
+        .. math::
```

Comment:
Do we have any way to avoid crossing the background border?

![image](https://user-images.githubusercontent.com/49409390/71732734-036f3780-2e16-11ea-9640-5f8e3ebbf390.png)

Reply:
Only options I can think of are:

1. Remove the page shadow (similar to the walrus documentation: https://the-walrus.readthedocs.io)

2. Somehow add a line break in that last line of the latex?

Reply:
Yeah I think we probably just need to be careful about equation width... but not worth sorting out now